manifests/bundle.yaml

```diff
@@ -72,7 +86,10 @@ spec:
         category: metrics-exporter
     spec:
       containers:
-      - image: storageos/metrics-exporter:v0.1.2
+      - args:
+        - -config
+        - /etc/storageos/metrics-exporter-config.yaml
```

Comment:
If the daemonset gets updated for these arguments on the cluster, the existing image on a node will crash unless the image is re-pulled. I guess it means we need to bump the release once again, is that correct?

Reply:
@Ricardo-Osorio pointed out that the operator will not fetch the changes from main, only from a tagged release. Which means `-config` will not be passed to an v0.1.5 image which does not understand it.